### DISCUSSION TITLE
Strapi NextJs Vercel

### DISCUSSION THREAD

**Discussion by JuanRodrigoVV** [vercel/community#1533] [Help]
### Summary

Hello everyone. I am having a problem with vercel strapi and nextjs. I have uploaded my projects yo railway and vercel. I can't get my deployed app to refresh the data when i change it in strapi.

**Comment by amyegan** [comment]
Hi @JuanRodrigoVV! I have a few questions to help guide us with debugging.
- Did you already [create a webhook in Strapi](https://strapi.io/blog/webhooks)?
- Do you see any error messages?
- What framework are you using for your project (e.g. Next.js, React, Svelte, Vue, etc)?
  - Is your site using SSR, SSG, or ISR?
